docker/fedora/Dockerfile

```diff
@@ -1,67 +1,39 @@
-FROM fedora:26
+FROM fedora:36
 LABEL maintainer="Dmitry Babokin <dmitry.y.babokin@intel.com>"
 SHELL ["/bin/bash", "-c"]
 
-ARG REPO=ispc/ispc
-ARG SHA=main
-
-# !!! Make sure that your docker config provides enough memory to the container,
-# otherwise LLVM build may fail, as it will use all the cores available to container.
+# TODO! fix to ispc and main
+ARG REPO=nurmukhametov/ispc
+ARG SHA=nurmukhametov/clang-cpp-lib
 
 # Packages required to build ISPC and Clang.
 # RUN echo "proxy=http://proxy.yourcompany.com:888" >> /etc/dnf/dnf.conf
-RUN dnf install -y vim-enhanced-2:8.0.1806 wget-1.19.5 yum-utils-1.1.31 gcc-7.3.1 gcc-c++-7.3.1 git-2.13.6 python3-3.6.4 && \
-    dnf install -y m4-1.4.18 bison-3.0.4 flex-2.6.1 patch-2.7.6 make-4.2.1 ncurses-devel-6.0 glibc-devel-2.25-13.fc26.x86_64 glibc-devel-2.25-13.fc26.i686 && \
-    dnf clean -y all
-
-# These packages are required if you need to link IPSC with -static.
-RUN dnf install -y ncurses-static-6.0 libstdc++-static-7.3.1 glibc-static-2.25 && \
+RUN dnf install -y vim-enhanced wget yum-utils git python3 cmake && \
```

Comment:
We need to have specific package versions, otherwise we have problems with security scans. @aneshlya know more about it.

Reply:
Security scans? Who or what performs them?

Reply:
Added versions. Fixed Readme.